test/test_prototype_transforms_consistency.py

```diff
@@ -422,34 +423,54 @@ def __init__(
         ],
         removed_params=["resample"],
     ),
+    ConsistencyConfig(
+        prototype_transforms.PILToTensor,
+        legacy_transforms.PILToTensor,
+    ),
+    ConsistencyConfig(
+        prototype_transforms.ToTensor,
+        legacy_transforms.ToTensor,
+    ),
+    ConsistencyConfig(
+        prototype_transforms.Compose,
+        legacy_transforms.Compose,
+    ),
+    ConsistencyConfig(
+        prototype_transforms.RandomApply,
+        legacy_transforms.RandomApply,
+    ),
+    ConsistencyConfig(
+        prototype_transforms.RandomChoice,
+        legacy_transforms.RandomChoice,
+    ),
+    ConsistencyConfig(
+        prototype_transforms.RandomOrder,
+        legacy_transforms.RandomOrder,
+    ),
+    ConsistencyConfig(
+        prototype_transforms.AugMix,
+        legacy_transforms.AugMix,
+    ),
+    ConsistencyConfig(
+        prototype_transforms.AutoAugment,
+        legacy_transforms.AutoAugment,
+    ),
+    ConsistencyConfig(
+        prototype_transforms.RandAugment,
+        legacy_transforms.RandAugment,
+    ),
+    ConsistencyConfig(
+        prototype_transforms.TrivialAugmentWide,
+        legacy_transforms.TrivialAugmentWide,
+    ),
 ]
 
 
 def test_automatic_coverage():
     available = {
         name
         for name, obj in legacy_transforms.__dict__.items()
-        if not name.startswith("_")
-        and isinstance(obj, type)
-        and not issubclass(obj, enum.Enum)
-        and name
-        not in {
```

Comment:
We no longer need an exclude list, since all transforms are now have a `ConsistencyConfig` :tada: